apps/wing/src/cli.ts

```diff
@@ -27,6 +39,33 @@ function actionErrorHandler(fn: (...args: any[]) => Promise<any>) {
   };
 }
 
+async function collectAnalyticsHook(cmd: Command) {
+  if (process.env.WING_DISABLE_ANALYTICS) { return; }
```

Comment:
If the user opted out of anlytics collection, it would be nice to show some feedback that no analytics are sent (maybe print to stdout)

Reply:
Let me think about how best to handle output info on analytics collection. I think this might become noise, maybe its 1. if disable AND if debug then we print the message.

Ill make suggestion in subsequent pr.